hoprd/rest-api/src/lib.rs

```diff
@@ -234,10 +224,33 @@ async fn build_api(
                     "/channels/:channelId/tickets/aggregate",
                     post(tickets::aggregate_tickets_in_channel),
                 )
+                .route("/messages", post(messages::send_message))
+                .route("/tickets/redeem", post(tickets::redeem_all_tickets))
+                .with_state(inner_state.clone().into())
+                .layer(middleware::from_fn_with_state(
+                    inner_state.clone(),
+                    preconditions::authenticate,
+                ))
+                .layer(middleware::from_fn_with_state(
```

Comment:
Please document why this group of routes needs the additional middleware, while the others don't. The more middleware we add the more complex it gets.